### PR TITLE
Bug/pdct 1168 unable to create new family on admin service

### PR DESCRIPTION
**Pull request by katybaulch**
# What's changed

- Removed organisation validation on family form (cause of bug)
- Corrected UNFCCC validation on family form
- Removed all references to watchOrganisation in family form
- Modify canModify function to enable if superuser and work for creating new families (cause of bug)

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [x] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

### CODE REVIEW

File: src/components/forms/FamilyForm.tsx [hunk 1]
```diff
@@ -945,11 +972,21 @@ export const FamilyForm = ({ family: loadedFamily }: TProps) => {
               )}
             </VStack>
 
-            <ButtonGroup isDisabled={!canModify(watchOrganisation, userAccess)}>
+            <ButtonGroup
+              isDisabled={
+                loadedFamily
+                  ? !canModify(
```
Comment (PatFawbertMills) [inline src/components/forms/FamilyForm.tsx:978]: We are calling the same method here with 2/3 of the same arguments, for readability could we change to something like: `!canModify(loadedFamily?.organisation || orgName, isSuperuser, userAccess)` ?

File: src/components/forms/FamilyForm.tsx [hunk 2]
```diff
@@ -140,60 +139,70 @@ export const FamilyForm = ({ family: loadedFamily }: TProps) => {
...
-    if (!token) return []
-    const decodedToken = decodeToken(token)
-    return decodedToken?.authorisation
+    if (!token) return null
+    const decodedToken: IDecodedToken | null = decodeToken(token)
+    return decodedToken
   }, [])
 
+  const userAccess = !userToken ? null : userToken.authorisation
+  const isSuperuser = !userToken ? false : userToken.is_superuser
```
Comment (PatFawbertMills) [inline src/components/forms/FamilyForm.tsx:180]: Please update to camel case `isSuperUser`

Comment (PatFawbertMills) [reply src/components/forms/FamilyForm.tsx:180]: Same in the `canModify` utility method

File: src/components/forms/FamilyForm.tsx (hunk 1 above)
Comment (katybaulch) [reply src/components/forms/FamilyForm.tsx:978]: I didn't realise you could do that! Super cool, doing now

**Review by PatFawbertMills — APPROVED** [review]
LGTM nice one

**Pull request merged**